docs/faq.rst

```diff
@@ -211,26 +211,26 @@ For making the **storage** highly available, all sorts of techniques can be used
 - btrfs in raid configuration
 - Distributed network filesystems such as GlusterFS or CEPH
 
-Note that no storage HA solution can protect against incidental deletes or file corruptions.
+Note that no storage HA solution can protect against accidental deletes or file corruptions.
 Therefore it is advised to create backups on a regular base!
 
 A backup MX can be configured as **failover**. For this you need a separate server running
 Mailu. On that server, your domains will need to be setup as "Relayed domains", pointing
-to you main server. MX records for the mail domains with a higher priority number will have
+to you mainr server. MX records for the mail domains with a higher priority number will have
 to point to this server. Please be aware that a backup MX can act as a `spam magnet`_.
 
 For **service** HA, please see: `How does Mailu scale up?`_
 
 
 *Issue reference:* `177`_, `591`_.
 
-.. _`spam magnet`: https://blog.zensoftware.co.uk/2012/07/02/why-we-tend-to-recommend-not-having-a-secondary-mx-these-days/
+.. _`spam magnet`: https://blog.zensoftware.co.uk/2012/07/02/why-we-tend-to-recommend-not-having-a-secondary-mx-these-days/ (archive.org)
```

Comment:
The link `https://blog.zensoftware.co.uk/2012/07/02/why-we-tend-to-recommend-not-having-a-secondary-mx-these-days/ ` works.

The changed link with (archive) does not work). Can you undo this?

Reply:
Fixed in a91e0a47eb480e0bd1822afa4148dbc068e46592
Prefer to use archive.org in case the blog post goes away.